ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -322,6 +322,11 @@ tests:
     env:
       GATEWAY_MODE: local
     workflow: openshift-e2e-aws-ovn-local-to-shared-gateway-mode-migration
+- as: e2e-aws-hypershift-ovn-kubevirt
+  optional: true
+  steps:
+    cluster_profile: aws
+    workflow: hypershift-kubevirt-conformance
```

Comment:
you might need the prerelease CNV versino for this job, you can enable that with an env var. 

    env:
      CNV_PRERELEASE_VERSION: "4.14" 

as seen in this file, `ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14__periodics.yaml`